lib/internal/process/next_tick.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const kMaxCallbacksPerTick = 1e4;
```

Comment:
Thanks for the feedback. I've changed the name of the const now and also added a comment.